lib/tasks/election.rake

```diff
@@ -27,4 +27,22 @@ namespace :election do
       )
     end
   end
+
+  task :end_ministerial_appointments, [:end_date] => :environment do |_t, args|
+    appointments = RoleAppointment.current.for_ministerial_roles
+    end_date = Date.today
+    prime_ministerial_role_id = 1
+    begin
+      if args[:end_date]
+        end_date = Date.parse(args[:end_date])
```

Comment:
What would the date input look like in the console? "YYYY-MM-DD"? Just wondering if this could cause user input errors. Could we add an example or some error handling? Or will it just be you running this?